certified-connectors/Data8/apiProperties.json

```diff
@@ -16,8 +16,10 @@
       }
     },
     "iconBrandColor": "#1973AE",
-    "capabilities": [],
+    "capabilities": [
+      "allowSharing": true
```

Comment:
@mattybeard can you please explain the reason for this "allowSharing" property?
"capabilities" should be an array of strings, and it could have two values "actions" and/or "triggers". Or you can leave it empty.

"allowSharing" property is used for a multi-auth connectors. please see [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/CognitiveServicesQuestionAnswering/apiProperties.json#:~:text=%22allowSharing%22%3A%20true)for example.

Reply:
Thanks for the reply @Amjed-Ayoub 
.
I'm currently in discussion with the Product Group around this feature as I want the connector to have it's connection within PowerApps shared at User level, not just Service Principal level and this is how I was advised. I wasn't however given the public schema, just the internal schema so was taking a bit of a guess at the schema but your link was helpful and I'll amend now.

@troystaylor has been involved in the same discussions.

Reply:
@mattybeard thank you so much for getting back to me. I will double check if it could be added to the main "properties" inside of a "metadata" property. Maybe this feature will be added internally to your connector by our team. I will get back to you as soon as possible.
Thank you so much again.